Sources/SnapshotTesting/Strategy/UIView.swift

```diff
@@ -0,0 +1,43 @@
+#if os(iOS) || os(tvOS)
+import UIKit
+
+extension Strategy where Snapshottable == UIView, Format == UIImage {
+  public static var image: Strategy {
+    return .image()
+  }
+
+  public static func image(
+    precision: Float = 1,
+    size: CGSize? = nil,
+    traits: UITraitCollection = .init()
+    )
+    -> Strategy {
+
+      return SimpleStrategy.image(precision: precision).asyncPullback { view in
+        snapshotView(
```

Comment:
This helper gets used for `UIView` and `UIViewController` to properly handle trait collection override, child view controller logic.